js/main.js

```diff
@@ -1,21 +1,135 @@
+const SIMILAR_PROPOSAL_COUNT = 10;
+
+const Author = {
+  AVATAR: getAvatarArray('img/avatars/user'),
+};
+
+const Offer = {
+  TITLE: 'Welcome to vacation',
+
+  ADDRESS: [
+    //LOCATION.lat,
+    //LOCATION.lat
+  ],
+
+  PRICE: {
+    min: 500,
+    max: 10000,
+  },
+
+  HOUSE_TYPE: ['palace', 'flat', 'house', 'bungalow', 'hotel'],
+
+  ROOMS: {
+    min: 1,
+    max: 20,
+  },
+
+  GUESTS: {
+    min: 1,
+    max: 60,
+  },
+
+  CHECKIN: ['12:00', '13:00', '14:00'],
+
+  CHECKOUT: ['12:00', '13:00', '14:00'],
+
+  FEATURES: ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'],
+
+  DESCRIPTION: 'Incredible location,lovely restaurant,friendly welcome, kindness staff',
+
+  PHOTOS: [
+    'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+    'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+    'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+  ],
+};
+
+const Location = {
+  LAT: {
+    min: 35.65,
+    max: 35.7,
+  },
+
+  LNG: {
+    min: 139.7,
+    max: 139.8,
+  },
+};
+
 function getRandomIntInclusive(min, max) {
   min = Math.ceil(min);
   max = Math.floor(max);
 
-  if(min < 0 || max <= min) {
+  if (min < 0 || max <= min) {
     return;
   }
 
   return Math.floor(Math.random() * (max - min + 1)) + min; //Максимум и минимум включаются
 }
-getRandomIntInclusive(5,4);
 
 function getRandomArbitrary(min, max) {
-
-  if(min < 0 || max <= min){
+  if (min < 0 || max <= min) {
     return;
   }
 
   return Math.random() * (max - min) + min;
 }
-getRandomArbitrary(3,6);
+
+function getMixArr(arr) {
```

Comment:
```suggestion
function getMixArray(array) {
```

нет смысла экономить на спичках и писать сокращённо, сокращает за тебя обфускатор и минимизатор, а тебе надо писать понятнее некуда, тк тебя читает человек.

Reply:
done